### DISCUSSION TITLE
Project structure

### DISCUSSION THREAD

**Discussion by Faumaray** [AutomateAbyss/Oxidesk#2] [General]
# Components List
It is necessary to decide which components to take out into separate libraries, in order to be able to use them for other purposes, for example, creating a desktop application based on the API.
There will definitely be a division into 3 mandatory components:

1. FrontEnd
2. BackEnd
3. Request\Response Messages

Next, tasks will be added to which a separate Issue or Project Task will be attached.
--------------------------------------------------------------------------------------------------------------------------------------------

**Comment by Faumaray** [comment]
@kam1xgod I summon you to discuss possible ideas

**Reply by kam1xgod** [reply]
I sleep.

**Comment by kam1xgod** [comment]
Even for "desktop application" there's a crucial need in API layer.
